test/e2e/testing-manifests/storage-csi/hostpath/hostpath/csi-hostpathplugin.yaml

```diff
@@ -33,7 +33,7 @@ spec:
           - mountPath: /registration
             name: registration-dir
         - name: hostpath
-          image: quay.io/k8scsi/hostpathplugin:v1.0.1
+          image: quay.io/k8scsi/hostpathplugin:canary
```

Comment:
need the latest changes from https://github.com/kubernetes-csi/csi-driver-host-path/blob/master/deploy/hostpath/csi-hostpath-plugin.yaml
e.g.
```
        - hostPath:
            path: /var/lib/kubelet/plugins
            type: Directory
            name: plugins-dir
```